packages/protocol-kit/src/Safe.ts

```diff
@@ -1698,6 +1700,15 @@ class Safe {
   }): ContractInfo | undefined => {
     return getContractInfo(contractAddress)
   }
+
+  /**
+   * This method creates a signer to be used with the init method
+   * @param {Credential} credential - The credential to be used to create the signer. Can be generated in the web with navigator.credentials.create
+   * @returns {PasskeyArgType} - The signer to be used with the init method
+   */
+  static createPasskeySigner = async (credential: Credential): Promise<PasskeyArgType> => {
+    return extractPasskeyData(credential)
+  }
```

Comment:
we should also update our documentation in safe-docs, to use `createPasskeySigner` instead of `extractPasskeyData`

Reply:
I already created a ticket for that